gobblin-runtime/src/test/java/org/apache/gobblin/runtime/spec_store/MysqlBaseSpecStoreTest.java

```diff
@@ -48,23 +48,23 @@ public class MysqlBaseSpecStoreTest {
   private static final String PASSWORD = "testPassword";
   private static final String TABLE = "base_spec_store";
 
+  private ITestMetastoreDatabase testDb;
```

Comment:
mostly this DB is NOT `static`, but sometimes it was.  for uniformity making it always instance-level

Reply:
although this is more rare w/ tests (however we DO have a few derived test classes), but anyway a `static` resource that needs closing doesn't really play well w/ subclassing